src/include/storage/copier/column_chunk.h

```diff
@@ -100,15 +104,31 @@ class ColumnChunk {
 class NullColumnChunk : public ColumnChunk {
 public:
     NullColumnChunk()
-        : ColumnChunk(common::LogicalType(common::LogicalTypeID::BOOL),
-              nullptr /* copyDescription */, false /* hasNullChunk */) {
-        resetNullBuffer();
```

Comment:
Why do we remove `resetNullBuffer` from constructor? Should it be called inside `NullColumnChunk::initialize()`.